docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.tt

```diff
@@ -0,0 +1,24 @@
+﻿<#@ include file="MulticlassClassification.ttinclude"#>
+<#+
+string ClassName="NaiveBayes";
+string Trainer = "NaiveBayes";
+string MetaTrainer = null;
+string TrainerOptions = null;
+
+string OptionsInclude = "";
+string Comments= "";
+
+string ExpectedOutputPerInstance= @"// Expected output:
+            //   Label: 1, Prediction: 2
```

Comment:
>Label: 1, Prediction: 2 [](start = 17, length = 23)

how come generated labels are 0,1,2 but here I see 1,2,3. how did they get changed?

Reply:
no, generated labels are 1,2,3

---
In reply to: [271938136](https://github.com/dotnet/machinelearning/pull/3159#discussion_r271938136) [](ancestors = 271938136)